dev/com.ibm.ws.sipcontainer.servlet.3.1/.project

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Should have .settings/bndtools.core.prefs and .settings/org.eclipse.jdt.core.prefs files for this new component.